code/modules/clothing/suits/toggles.dm

```diff
@@ -0,0 +1,64 @@
+/obj/item/clothing/suit/wintercoat
+    name = "winter coat"
+    desc = "A warm, fluffy and incredibly comfortable jacket made from animal furs."
+    icon_state = "coatwinter"
+    var item_state_slots = list(slot_r_hand_str = "coatwinter", slot_l_hand_str = "coatwinter")
+    body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+    heat_conductivity = SPACESUIT_HEAT_CONDUCTIVITY
+    armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 10, rad = 0)
+//    action_button_name = "Toggle Hood"
+    allowed = list (
+    /obj/item/weapon/pen,
+    /obj/item/weapon/paper,
+    /obj/item/device/flashlight,
+    /obj/item/weapon/tank/emergency_oxygen,
+    /obj/item/weapon/storage/fancy/cigarettes,
+//    /obj/item/weapon/storage/box/matches,
+    /obj/item/weapon/reagent_containers/food/drinks/flask
+    )
+
+
+/obj/item/clothing/suit/wintercoat/cargo
+    name = "cargo technicians winter coat"
```

Comment:
Don't like this name, doesn't flow, and is grammatically incorrect without an apostrophe